src/main/java/datagateway/event/EventToEventReader.java

```diff
@@ -8,9 +8,13 @@
 
 public class EventToEventReader implements EventReader{
     private final Event event;
+    private final String name;
+    private final boolean completed;
 
-    public EventToEventReader(Event event){
+    public EventToEventReader(Event event, String name, boolean completed){
         this.event = event;
+        this.name = name;
+        this.completed = completed;
```

Comment:
why shouldn't we obtain the event name and completion from the event itself?

Reply:
Event doesn't have this info anymore since it doesn't explicitly store a task. It stores the taskId which is used to get the name and completed status.

Reply:
it would make sense for event to have its own name, separated from task. e.g. you might have "complete tutorial worksheet" as your task, and "meeting with study group to discuss tutorial problems" as your event. but i guess its a small thing and we dont need to make the change.